module/extractor/download_iterator.py

```diff
@@ -11,8 +11,7 @@ def download_and_save_iterator(r, chunk_size, total_size, title, isGUI=False, wi
                 download_percentage = ((chunk_size * (curr_chunk + 1)) / total_size) * 100
                 window[element['progress_bar_key']].update(download_percentage)
                 window[element['percent_key']].update(download_percentage)
-
-            f.write(data)
+                f.write(data)
```

Comment:
Secondly I have created a branch for the GUI, so please make a new pr to that branch for me when you do with change please.
Lastly, do you know about the module py2exe or pyinstaller, generate single .exe file

I need your email or whatsapp for future collaboration 🥂

Thank you

Reply:
I have never worked with either of those modules before. My email is: bernausergi@gmail.com